### ISSUE TITLE
CI a11y test step will pass even when a11y report shows failures

### ISSUE THREAD

**Issue by kmcfaul** [patternfly/patternfly-react#11416]
Can be seen on various PRs, for example https://github.com/patternfly/patternfly-react/pull/11377 with its a11y report https://patternfly-react-pr-11377-a11y.surge.sh/

The report shows 4 failing tests, but the CI has passed. The CI should fail when there are a11y issues.

**Comment by mfrances17** [comment]
This was an issue in how the patternfly-a11y tests are processing error codes, so CI was not picking them up accordingly. I put up a PR in patternfly-a11y here:

https://github.com/patternfly/patternfly-a11y/pull/87
